partner_default_sale_discount/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
Please change license lines in this module to https:

Reply:
Done.